.github/workflows/pypi.yml

```diff
@@ -18,6 +18,15 @@ jobs:
         pip install -r requirements-dev.txt
     - name: Test
       run: make test
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v2
+      with:
+        directory: ./coverage/reports/
+        env_vars: OS,PYTHON
+        fail_ci_if_error: true
+        files: ./coverage.xml
+        flags: unittests
+        verbose: true
```

Comment:
If you have added the `test_cov.yml` workflow to upload the report, so it should not exist in `pypi` workflow, how do you think about it？